Änderung in der Blizzbot.py

flloschy opened the pull request:
Der Bot reagiert nicht mehr in DMs, um mögliche Error Meldungen zu umgehen falls in der DM ein Blacklist Wort geschrieben wurde

Wenn eine Blacklist wort gefunden wurde wird Überprüft ob die Nachricht zu lang ist, wenn ja wird nicht der Nachrichten Content gesendet um einen fehler zu um gehen da die Nachricht zu lang ist

flloschy committed:
Änderung in der on_message def

Der Bot reagiert nicht mehr in DMs, um mögliche Error Meldungen zu umgehen falls in der DM ein Blacklist Wort geschrieben wurde

Wenn eine Blacklist wort gefunden wurde wird Überprüft ob die Nachricht zu lang ist, wenn ja wird nicht der Nachrichten Content gesendet um einen fehler zu um gehen da die Nachricht zu lang ist

CrispiCas commented on blizzbot.py:
```diff
@@ -71,42 +71,49 @@ async def anfrage(ctx):
+                else:
+                    await message.author.dm_channel.send("Ihre Nachricht wurde entfernt. Melden Sie sich bei Fragen an einen Moderator.")
+                except:
+                pass
+            
```
Pass ist nicht eingerückt...

CrispiCas commented on blizzbot.py:
```diff
@@ -71,42 +71,49 @@ async def anfrage(ctx):
-
-    if message.guild and message.author != bot.user and message.channel.id != IDchanneladmin:
-        if await zz_functions.checkwords(message):
+                try:
+                await message.author.create_dm()
+                if len(message.content) > 1500:
+                    await message.author.dm_channel.send(f"Ihre Nachricht mit dem Inhalt **{message.content}** wurde entfernt. Melden Sie sich bei Fragen an einen Moderator.")
+                else:
+                    await message.author.dm_channel.send("Ihre Nachricht wurde entfernt. Melden Sie sich bei Fragen an einen Moderator.")
```
Nach dem Try ist auch nicht eingerückt...

flloschy committed:
Update blizzbot.py

Pass ist jetzt eingedrückt, so wie try :)
Ups

flloschy committed:
Log Schutz

Da Embed fields nur 1028 Zeichen haben können kann es zu Fehlern kommen, oder dazu das Nachrichten nicht geloggt werden, deswegen ein kleiner Schutz
